includes/elements/section.php

```diff
@@ -434,8 +434,7 @@ protected function _register_controls() {
 			]
 		);
 
-		$is_legacy_mode_active = Plugin::instance()->get_legacy_mode( 'elementWrappers' );
-		$content_position_selector = $is_legacy_mode_active ? '{{WRAPPER}} > .elementor-container > .elementor-row > .elementor-column > .elementor-column-wrap > .elementor-widget-wrap' : '{{WRAPPER}} > .elementor-container > .elementor-column > .elementor-widget-wrap';
+		$content_position_selector = Plugin::$instance->experiments->is_feature_active( 'e_dom_optimization' ) ? '{{WRAPPER}} > .elementor-container > .elementor-column > .elementor-widget-wrap' : '{{WRAPPER}} > .elementor-container > .elementor-row > .elementor-column > .elementor-column-wrap > .elementor-widget-wrap';
```

Comment:
Make it more readable.
```suggestion
		$content_position_selector = Plugin::$instance->experiments->is_feature_active( 'e_dom_optimization' )
		? 
		'{{WRAPPER}} > .elementor-container > .elementor-column > .elementor-widget-wrap' 
		:
		 '{{WRAPPER}} > .elementor-container > .elementor-row > .elementor-column > .elementor-column-wrap > .elementor-widget-wrap';
```